core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/Server.json

```diff
@@ -1171,8 +1171,7 @@
                         "LoggingEnabled":       [ {"version": "[10,)",         "wlst_mode": "both",    "wlst_name": "LoggingEnabled",       "wlst_path": "WP001", "value": {"default": "true"                                }, "wlst_type": "boolean"  } ],
                         "Notes":                [ {"version": "[10,)",         "wlst_mode": "both",    "wlst_name": "Notes",                "wlst_path": "WP001", "value": {"default": "None"                                }, "wlst_type": "string"   } ],
                         "NumberOfFilesLimited": [ {"version": "[10,)",         "wlst_mode": "both",    "wlst_name": "NumberOfFilesLimited", "wlst_path": "WP001", "value": {"default": "true"                                }, "wlst_type": "boolean"  } ],
-                        "RotateLogOnStartup":   [ {"version": "[10,12.2.1.3)", "wlst_mode": "both",    "wlst_name": "RotateLogOnStartup",   "wlst_path": "WP001", "value": {"default": "true"                                }, "wlst_type": "boolean"  } ,
-                                                  {"version": "[12.2.1.3,)",   "wlst_mode": "both",    "wlst_name": "RotateLogOnStartup",   "wlst_path": "WP001", "value": {"default": "${true:false}"                       }, "wlst_type": "boolean"  } ],
+                        "RotateLogOnStartup":   [ {"version": "[10,)",         "wlst_mode": "both",    "wlst_name": "RotateLogOnStartup",   "wlst_path": "WP001", "value": {"default": "true"                                }, "wlst_type": "boolean"  } ],
```

Comment:
This seems like the opposite of what would fix the filed issue.

Reply:
The default setting of false would have a mismatch between the offline and online in createdomain

Reply:
This needed to be fixed for discover but you ar right it doesn't fix this issue. I cannot recreate the issue